### DISCUSSION TITLE
Forgot my username/password

### DISCUSSION THREAD

**Discussion by hello-nico-chan** [datalust/seq-tickets#1850] [Help] [answered]
Hello team, I installed Seq on a Windows Server machine. Now I cannot remember what my username/password is, and I tried this:
``` shell
seq stop
seq auth --basic -u Admin -p <new_password>
```
but returned:
``` shell
Usage: seq <command> [<args>]
Type 'seq help' for available commands
```

`seq unlock` is same.

Whole shell codes:
``` shell
PS C:\Program Files\Seq seq stop
Stopping Seq...
Waiting up to 60 seconds for apps to unload and the service to stop (currently: Running)...
Stopped.
PS C:\Program Files\Seq seq auth --basic -u Admin -p <new_password>
usage: seq <command> [<args>]
Type 'seq help' for available commands
PS C:\Program Files\Seq
```

What should I do?

Thanks!

**Comment by liammclennan** [comment, marked as answer]
Hi @hello-nico-chan 

Please see https://docs.datalust.co/docs/recovering-from-broken-authentication-settings. You may need to choose the version of the documentation that corresponds to your version of Seq.

**Reply by hello-nico-chan** [reply]
Oops, I need to use `seq auth **reset** -u admin -p <new_password>`, not `seq auth **--basic** -u admin -p <new_password>`. Thank you so much @liammclennan !!!

**Reply by liammclennan** [reply]
You're welcome.
